packages/ui-signer/src/types.d.ts

```diff
@@ -9,52 +9,52 @@ import { Base, UInt } from '@polkadot/types/codec';
 import { Hash, Extrinsic } from '@polkadot/types';
 import { RawParam$Value } from '@polkadot/ui-app/Params/types';
 
-export type QueueTx$Status = 'finalised' | 'usurped' | 'dropped' | 'broadcast' | 'cancelled' | 'completed' | 'error' | 'incomplete' | 'queued' | 'sending' | 'sent';
+export type AccountInfo = {
+  accountId?: string | null,
+  accountNonce: UInt | BN
```

Comment:
Not a major thing, but since we are fiddling here, it can now just be `accountNonce: BN` (Uint now extends BN, so no need to import that anymore)